charts/histogram/src/ext.js

```diff
@@ -36,12 +38,16 @@ export default function ext(env) {
       });
     },
     exportProperties(propertyTree, hyperCubePath, viewDataMode) {
-      return conversion.hypercube.exportProperties({ propertyTree, hyperCubePath, viewDataMode });
+      return conversion.hypercube.exportProperties({
+        propertyTree,
+        hyperCubePath: viewDataMode ? VIEW_DATA_HYPERCUBE_PATH : hyperCubePath,
+        viewDataMode,
+      });
     },
     requireNumericDimension: true,
     exportTableProperties(propertyTree) {
       // the hypercube that will be used for "building" the table is under `qUndoExclude.box.qHyperCubeDef`
-      const hypercubePath = 'qUndoExclude.box';
+      const hypercubePath = VIEW_DATA_HYPERCUBE_PATH;
       return conversion.hypercube.exportProperties({ propertyTree, hypercubePath });
```

Comment:
should `viewDataMode: true` be passed into `conversion.hypercube.exportProperties` as well? Not really sure what it does, just comparing with what you did for the boxplot above :)

Reply:
Yes, we need to pass `viewDataMode: true` here as you said @veinfors.
viewDataMode originally is for modifiers where we don't want to show generated expression by a modifier, but now it is extended to use for what we want to see in the view data mode.